src/main/java/net/sf/jsqlparser/expression/MySqlSqlCalcFoundRows.java

```diff
@@ -0,0 +1,33 @@
+/*
+ * #%L
+ * JSQLParser library
+ * %%
+ * Copyright (C) 2004 - 2017 JSQLParser
+ * %%
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Lesser General Public License as
+ * published by the Free Software Foundation, either version 2.1 of the
+ * License, or (at your option) any later version.
+ * 
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Lesser Public License for more details.
+ * 
+ * You should have received a copy of the GNU General Lesser Public
+ * License along with this program.  If not, see
+ * <http://www.gnu.org/licenses/lgpl-2.1.html>.
+ * #L%
+ */
+package net.sf.jsqlparser.expression;
+
+/**
+ * @author sam
```

Comment:
Could you add the reference link here? And that this object supports: `SQL_CALC_FOUND_ROWS` for the MySQL driver?

Reply:
Yes. SQL_CALC_FOUND_ROWS keyword is hint syntax only for MySQL driver.
You can find here. 
https://dev.mysql.com/doc/refman/5.7/en/select.html

Reply:
Mind adding this to the doc:
```
Supports the {@code SQL_CALC_FOUND_ROWS} keyword for <a href="https://dev.mysql.com/doc/refman/5.7/en/select.html">MySQL driver.</a>
```

Just so there's a reference in case future devs want to know more about what this does (then they don't have to track this PR down) :)